src/eth/primitives/index.rs

```diff
@@ -1,17 +1,41 @@
+use std::ops::Add;
+
+use ethereum_types::U256;
+use ethereum_types::U64;
 use sqlx::database::HasValueRef;
 use sqlx::error::BoxDynError;
 
 use crate::gen_newtype_from;
 
-#[allow(dead_code)]
-/// Representation of a transaction index or log index.
+/// Index is a representation for transaction indexes and log indexes
```

Comment:
Some practices from the Rust community when writing docs:
* No need to repeat the struct name or have phrases like "This is struct" or "This method" because documentation is contextual.
* Also all phrases should ends with a period.
* Verbs are always written as third-person singular.

Confirm this by checking the documentation of popular repositories.

Reply:
thanks!